### PR TITLE
More UI fixes and improvements

### PR DESCRIPTION
**Pull request by liamhays**
# What's new

- Multiple apps: "Exit to ... Menu?" and "Retry Reading?" are now uniformly placed and spelled in every app. "All unsaved data will be lost!" is also uniformly formatted in every app.
- NFC: For Ultralight/NTAG cards, "Unlock from Reader" is only displayed in the "Unlock with Password" menu instead of in both that menu and the main card menu.
- BadUSB: "Keyboard Layout", not "Keyboard layout".
- iButton: Awkward "Waiting for key ..." replaced with "Apply key to Flipper's back"
- Text input dialog: Auto capitalization now works (matching expected behavior) if text is selected and a letter is pressed
- NFC: "Detect Reader" instead of "Detect reader" in Mifare Classic saved menu
- NFC: Capitalization of prepositions is consistent with the rest of the OS

# Verification 

- Compile and upload to device and browse menus. To test the text input dialog, create a new file in any app and save it to a custom name.

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/main/nfc/scenes/nfc_scene_saved_menu.c [hunk 1]
```diff
@@ -75,21 +75,15 @@ void nfc_scene_saved_menu_on_enter(void* context) {
...
         submenu_add_item(
             submenu,
-            "Unlock With Reader",
-            SubmenuIndexMfUlUnlockByReader,
-            nfc_scene_saved_menu_submenu_callback,
-            nfc);
-        submenu_add_item(
-            submenu,
-            "Unlock With Password",
+            "Unlock",
```
Comment (skotopes) [inline applications/main/nfc/scenes/nfc_scene_saved_menu.c]: Something went wrong here. Probably merge artifacts, please fix.

File: applications/main/nfc/scenes/nfc_scene_saved_menu.c [hunk 2]
```diff
@@ -161,9 +155,9 @@ bool nfc_scene_saved_menu_on_event(void* context, SceneManagerEvent event) {
         } else if(event.event == SubmenuIndexRestoreOriginal) {
             scene_manager_next_scene(nfc->scene_manager, NfcSceneRestoreOriginalConfirm);
             consumed = true;
-        } else if(event.event == SubmenuIndexMfUlUnlockByReader) {
+	    /*} else if(event.event == SubmenuIndexMfUlUnlockByReader) {
```
Comment (skotopes) [inline applications/main/nfc/scenes/nfc_scene_saved_menu.c]: Why?

File: applications/services/gui/modules/text_input.c [hunk 3]
```diff
@@ -305,13 +305,14 @@ static void text_input_handle_right(TextInput* text_input, TextInputModel* model
...
 
+// TODO: fix auto capitalization
 static void text_input_handle_ok(TextInput* text_input, TextInputModel* model, bool shift) {
     char selected = get_selected_char(model);
     size_t text_length = strlen(model->text_buffer);
 
-    bool toogle_case = text_length == 0;
-    if(shift) toogle_case = !toogle_case;
-    if(toogle_case) {
+    bool toggle_case = (text_length == 0) || (model->clear_default_text);
```
Comment (skotopes) [inline applications/services/gui/modules/text_input.c]: Should be in separate PR.

**Review by skotopes — CHANGES_REQUESTED** [review]

**Comment by skotopes** [comment]
Please un-draft when ready

File: applications/main/nfc/scenes/nfc_scene_saved_menu.c (hunk 1 above)
Comment (liamhays) [reply applications/main/nfc/scenes/nfc_scene_saved_menu.c]: I'm not a UI designer and I shouldn't have made such drastic changes. I updated it so that the only changes are in spelling, to match capitalization across the rest of the OS.

File: applications/services/gui/modules/text_input.c (hunk 3 above)
Comment (liamhays) [reply applications/services/gui/modules/text_input.c]: Reverted, will make new PR.

File: applications/main/nfc/scenes/nfc_scene_saved_menu.c (hunk 2 above)
Comment (liamhays) [reply applications/main/nfc/scenes/nfc_scene_saved_menu.c]: See other NFC comment.

File: applications/main/ibutton/scenes/ibutton_scene_exit_confirm.c [hunk 4]
```diff
@@ -19,9 +19,9 @@ void ibutton_scene_exit_confirm_on_enter(void* context) {
     widget_add_button_element(
         widget, GuiButtonTypeRight, "Stay", ibutton_scene_exit_confirm_widget_callback, ibutton);
     widget_add_string_element(
-        widget, 64, 19, AlignCenter, AlignBottom, FontPrimary, "Exit to iButton menu?");
+        widget, 64, 19, AlignCenter, AlignBottom, FontPrimary, "Exit to iButton Menu?");
     widget_add_string_element(
-        widget, 64, 31, AlignCenter, AlignBottom, FontSecondary, "All unsaved data will be lost!");
+        widget, 64, 31, AlignCenter, AlignBottom, FontSecondary, "All unsaved data\nwill be lost!");
```
Comment (skotopes) [inline applications/main/ibutton/scenes/ibutton_scene_exit_confirm.c]: image

Also check all other places where `widget_add_string_element` used with `\n` they got same issue

**Review by skotopes — CHANGES_REQUESTED** [review]

**Comment by skotopes** [comment]
Un-draft when ready

**Review by skotopes — APPROVED** [review]
